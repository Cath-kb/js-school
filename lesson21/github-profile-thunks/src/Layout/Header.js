import React from "react";

const Header = ({title}) => (
  <header className="header">
    <h1 className="title text-center mb-4">{title}</h1>
  </header>
);

export default Header;
