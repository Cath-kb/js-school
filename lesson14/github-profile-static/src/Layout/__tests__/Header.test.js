import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from './../Header';

describe('Header', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header title="test" />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should be rendered without errors', () => {
    const html = renderer.create(<Header title="test" />).toJSON();
    expect(html).toMatchSnapshot();
  });
});
