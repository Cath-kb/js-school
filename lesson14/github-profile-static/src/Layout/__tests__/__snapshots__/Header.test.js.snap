// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should be rendered without errors 1`] = `
<header
  className="header"
>
  <h1
    className="title text-center mb-4"
  >
    test
  </h1>
</header>
`;
